composer.json

```
@@ -4,7 +4,7 @@
     "license": "MIT",
     "type": "symfony-bundle",
     "require": {
-        "php": ">=7.4",
+        "php": ">=8.0",
```

```suggestion
        "php": ">=8.1",
```

Let's just go with 8.1